src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Ball {
+    private final Colors color;
+    private final int number;
+
+    public Ball() {
+        this.color = new ColorSupplier().getRandomColor();
+        this.number = new NumberSupplier().getRandomNumber();
```

Comment:
You should do this not in the constructor, but in the getRandomBall method of the Lottery class